app/server/utils/user.py

```diff
@@ -592,34 +601,24 @@ def send_phone_verification_message(to_phone, one_time_code):
         send_message(to_phone, reciever_message)
 
 
-def change_pin(user: User, new_pin):
-    user.pin = new_pin
-    db.session.commit()
-    send_pin_change_messages(user, new_pin)
+def send_sms(user, message_key):
+    message = i18n_for(user, "ussd.kenya.{}".format(message_key))
```

Comment:
this helper function doesn't make sense in utils/user since the key is specific to ussd.kenya

Reply:
Really getting sloppy here 🤦🤦. Cleanup should be up in 30-45 tops currently in transit.